marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/mastering/previewMatchingActivity2/setup.xqy

```diff
@@ -0,0 +1,33 @@
+xquery version "1.0-ml";
+import module namespace hub-test = "http://marklogic.com/data-hub/test" at "/test/data-hub-test-helper.xqy";
+hub-test:reset-hub()
+
+;
+
+xquery version "1.0-ml";
+import module namespace hub-test = "http://marklogic.com/data-hub/test" at "/test/data-hub-test-helper.xqy";
+import module namespace test = "http://marklogic.com/test" at "/test/test-helper.xqy";
+hub-test:load-entities($test:__CALLER_FILE__)
+
+;
+
+xquery version "1.0-ml";
+import module namespace hub-test = "http://marklogic.com/data-hub/test" at "/test/data-hub-test-helper.xqy";
+import module namespace test = "http://marklogic.com/test" at "/test/test-helper.xqy";
+hub-test:load-artifacts($test:__CALLER_FILE__)
+
+;
+
+xquery version "1.0-ml";
+import module namespace test = "http://marklogic.com/test" at "/test/test-helper.xqy";
+import module namespace const = "http://marklogic.com/smart-mastering/constants"
+  at "/com.marklogic.smart-mastering/constants.xqy";
+for $uri in ("/dictionary/last-names.xml", "/thesaurus/nicknames.xml")
```

Comment:
If CONTENT-COLL doesn't actually matter for functionality, then you can get both of these docs loaded by tossing them under test-data/content .